test/integration-tests/language/LanguageClientIntegration.test.ts

```diff
@@ -42,11 +46,22 @@ suite("Language Client Integration Suite @slow", function () {
         async setup(ctx) {
             workspaceContext = ctx;
 
+            const resetSettings = await updateSettings({
+                "swift.sourcekit-lsp.serverArguments": [
+                    "--experimental-feature",
+                    "synchronize-request",
+                ],
```

Comment:
Instead of adding the to specify this as part of `initializationOptions` during the `initialize` request? As a general direction, I would like to move away from taking command line arguments to SourceKit-LSP.

Reply:
What does that look like? Adding `"workspace/synchronize-request": true` to the options here? https://github.com/swiftlang/vscode-swift/blob/main/src/sourcekit-lsp/LanguageClientManager.ts#L650

Reply:
It should be adding `"experimentalFeatures": ["synchronize-request"]` there. Though it would be nice to only enable that experimental feature for tests that need it. If that’s not possible, let me know and we can try to prioritize making the request non-experimental.

Reply:
https://github.com/swiftlang/vscode-swift/pull/1463